apps/web/.env.example

```diff
@@ -15,3 +15,7 @@ FEATURE_SKILL_LIBRARY=true
 LOG_CONSOLE_LEVEL="debug"
 LOG_APPLICATIONINSIGHTS_LEVEL="warning"
 
+# Maintenance banner
```

Comment:
Could you please add the example values (like from the PR) as comments?  For the duration, please note hours as the unit.

Reply:
Nice catch 👍 Fixed by [5a203ec](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7832/commits/5a203eca2ec6a1a9413f2720371b60a70da05198)